code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -462,7 +462,7 @@ var/global/list/damage_icon_parts = list()
 		//if(!src.get_int_organ(/obj/item/organ/internal/brain) && src.get_species() != "Machine" )//make it obvious we have NO BRAIN
 		//	hair_standing.Blend(debrained_s, ICON_OVERLAY)
 		if(hair_style && hair_style.species_allowed)
-			if(src.species.name in hair_style.species_allowed)
+			if((src.species.name in hair_style.species_allowed) | (src.species.flags & ALL_RPARTS))

```

Comment:
Got it, thanks. I've always been confused about when to use the bitwise stuff, but this helped clear it up some